examples/rosettacode/bull_and_cow.nit

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: bulls and cows
+# SEE: <http://rosettacode.org/wiki/Bulls_and_cows
+#
+
+# Retuen an array of length 4 with unique random number.
+fun random : Array[Char] do
+	var tab = new Array[Char]
+	while tab.length < 4 do
+		var element = 10.rand.to_c
+		if not tab.has(element) then tab.add(element)
+	end
+return tab
```

Comment:
You can use this:
~~~Nit
var tab = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
tab.suffle
return tab.subarray(0, 4)
~~~